src/media/css/site.styl

```diff
@@ -257,6 +257,12 @@ a:active {
     }    
 }
 
+@media $wide-tablet {
+    .subheader.hidden {
```

Comment:
This is the right approach but we shouldn't be calling something hidden yet give it instructions to show. It's fine to just hide the subheader on this page (try [data-page-type~=...]). Also I get the feeling you'll need to trigger the media query on `$base-tablet` instead. You're on the right path.